scripts/release

```diff
@@ -7,16 +7,6 @@ DEPLOY="${DEPLOY:-false}"
 source $(dirname $0)/lib/debug_functions
 source $(dirname $0)/lib/version
 
-
-# This flag is passed from travis, so only the right jobs will deploy the
-# container images to quay
-
-if [ "$DEPLOY" != "true" ] ; then
-    echo "We don't need to deploy from this job, if you're trying to deploy manually set DEPLOY=true"
-    exit 0
-fi
-
-
```

Comment:
This logic was removed in shipyard/scripts/release, so it will go away here when we move to Shipyard. I think we don't need this PR anymore.